### PR TITLE
test: add tests for stream chat flutter

### PR DESCRIPTION
**Pull request by deven98**
Added UI tests

### CODE REVIEW

**Comment by imtoori** [comment]
@ROCKSTARfromday1 take a look at our website to get to know how you can use our service for your purposes :) https://getstream.io/

File: packages/stream_chat_flutter/lib/src/attachment_actions_modal.dart [hunk 1]
```diff
@@ -145,19 +145,18 @@ class AttachmentActionsModal extends StatelessWidget {
...
                         final channel = StreamChannel.of(context).channel;
                         if (message.attachments.length > 1 ||
-                            message.text.isNotEmpty) {
+                            message.text?.isNotEmpty == true) {
                           final remainingAttachments = [...message.attachments]
                             ..removeAt(currentIndex);
                           channel.updateMessage(message.copyWith(
                             attachments: remainingAttachments,
                           ));
                           Navigator.pop(context);
```
Comment (Nash0x7E2) [inline packages/stream_chat_flutter/lib/src/attachment_actions_modal.dart:154]: You can cascade these methods :)

File: packages/stream_chat_flutter/test/src/attachment_widgets_test.dart [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            ),
+          ),
+        ),
+      );
+
+      expect(find.text('example.pdf'), findsOneWidget);
+    },
+  );
+
+  // testWidgets(
```
Comment (Nash0x7E2) [inline packages/stream_chat_flutter/test/src/attachment_widgets_test.dart:51]: Consider removing unused code :)

File: packages/stream_chat_flutter/test/src/channel_header_test.dart [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+      expect(find.byType(ChannelInfo), findsNothing);
+      expect(find.text('leading'), findsOneWidget);
+      expect(find.text('title'), findsOneWidget);
+      expect(find.text('subtitle'), findsOneWidget);
+      expect(find.text('action'), findsOneWidget);
+    },
+  );
+
+  testWidgets(
+    'shoBackButton: false should hide the StreamBackButton and '
```
Comment (Nash0x7E2) [inline packages/stream_chat_flutter/test/src/channel_header_test.dart:256]: Typo

File: packages/stream_chat_flutter/test/src/message_text_test.dart [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                  message: Message(
+                    text: 'demo',
+                  ),
+                  messageTheme: streamTheme.otherMessageTheme),
+            ),
+          ),
+        ),
+      ));
+
+      // TODO: Add better test
```
Comment (Nash0x7E2) [inline packages/stream_chat_flutter/test/src/message_text_test.dart:51]: How goes this?

**Review by Nash0x7E2 — CHANGES_REQUESTED** [review]
Consider using setup and teardown for defining mocks

**Pull request merged**